_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -229,80 +234,124 @@ These two HTML `a` elements have the same [accessible name][] and [context][prog
+			<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/admissions/contact.html">Contact Us</a>
+		</span>
+	</div>
+</html>
+```
+
+#### Failed Example 4
+
 These two HTML `span` elements have an [explicit role][] of link, same [accessible name][] and [context][programmatically determined link context], but link to resources that offer different content.
 
 ```html
 <html lang="en">
-	<p>
-		Learn more (<span
+	<div>
+		<span style="text-align:center;">Contact us</span>
+		<span style="display:flex; justify-content:space-around;">
+			<img src="/test-assets/shared/chat.png" alt="Chat" style="width:50%;">
+			<img src="/test-assets/shared/phone.png" alt="Phone" style="width:50%;">
+		</span>
+		<span style="display:flex; justify-content:space-around;">
+			<span
 			role="link"
 			tabindex="0"
-			onclick="location='/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html'"
-			>Contact us</span
-		>) and get in touch (<span
+			onclick="location='/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html'">Contact Us</span>
+			<span
 			role="link"
 			tabindex="0"
-			onclick="location='/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/admissions/contact.html'"
-			>Contact us</span
-		>)
-	</p>
+			onclick="location='/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/admissions/contact.html'">Contact Us</span>
+		</span>
+	</div>
 </html>
 ```
 
-#### Failed Example 4
+#### Failed Example 5
 
 These two SVG `a` elements have the same [accessible name][] and [context][programmatically determined link context] but link to different resources.
 
 ```html
 <html lang="en">
 	<p>
-		<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-			<a href="https://act-rules.github.io/" aria-label="ACT rules">
-				<circle cx="50" cy="40" r="35" />
+		<svg enable-background="new 0 0 264 120" viewBox="0 -20 264 140" xmlns="http://www.w3.org/2000/svg">
+			<text>Contact us</text>
+			<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/contact-us.html?page=1" aria-label="Contact Us">
+				<path d="m212.0806 68.0717c-10.3917 10.3852-22.4311 20.3239-27.1905 15.5646-6.8075-6.8075-11.0088-12.7418-26.0285-.6696-15.0132 12.0657-3.4792 20.1139 3.1182 26.7047 7.6149 7.6149 36.0001.407 64.0571-27.6434 28.0504-28.057 35.2386-56.4422 27.6172-64.0571-6.5974-6.604-14.6062-18.1314-26.6719-3.1182-12.0723 15.0132-6.1444 19.2145.6761 26.0285 4.7397 4.7593-5.1925 16.7988-15.5777 27.1905z"/>
 			</a>
-
-			<a href="https://www.w3.org/community/act-r/">
-				<text x="50" y="90" text-anchor="middle">
-					ACT rules
-				</text>
+			<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/contact-us.html?page=2" aria-label="Contact Us">
```

Comment:
It seems the "contact us" page doesn't act on `?page=1/2` but only on the more complex values.

Reply:
Uhm... it should thanks to 
`var page = urlParams.get('page');`

Are you sure it doesn't work? I'll check again.

Reply:
😓 Haven't followed that bit of the logic. Then it is probably OK…

Reply:
Yes, it should indeed work . Sorry about that.